### ISSUE TITLE
Logic tools plotting artefact

### ISSUE THREAD

**Issue by DanielGuramulta** [analogdevicesinc/scopy#972]
> * There is an plotting artifact when plotting lare buffersizes
> 
> ![image](https://user-images.githubusercontent.com/8676912/97871493-41cad100-1d1d-11eb-9b36-fd6f2f40c6e0.png)

**Comment by DanielGuramulta** [comment]
Fixed with: #971

**Comment by tagoylo** [comment]
Not sure if this is related, but this is what it looks like when sample rate is decreased. Are the lines considered the artifacts?
![ice_video_20210428-154218_edit_0](https://user-images.githubusercontent.com/25096080/116367943-6d48f700-a83a-11eb-98b2-645852ee8a0d.gif)

**Comment by adisuciu** [comment]
Yes, they are artifacts, however, as opposed to the previous version, the plotting should now be correct. I'll keep investigating the artifact
